presto-docs/src/main/sphinx/release/release-0.260.1.rst

```diff
@@ -0,0 +1,8 @@
+===============
+Release 0.260.1
+===============
+
+General Changes
+_______________
+* Fix a concurrent issue which could cause query failures. (:pr:`16664`)
+* Disable the AGG IF to FILTER rewrite rule by default as it could cause query failures in some corner cases. (:pr:`16661`)
```

Comment:
If we could be more specific on "some" or "could" part, that will help with our customers to understand the impact. Would love to see @mayankgarg1990's comments on this.

Reply:
Added more details to the note to explain which cases could trigger the issues.